approval_polls/static/approval_polls/poll_management.js

```diff
@@ -0,0 +1,31 @@
+$(function () {
+    var id = $(this).closest('div').attr('id');
+  $('.show-alert').click(function() {
+      $('#alert'+$(this).attr('id')).show();
+  });
+  
+  $('.hide-alert').click(function() {
+      $('#alert'+$(this).attr('id')).hide();
+  });
+  
+  $('.delete-poll').click(function() {
+	var id = $(this).attr('id');
```

Comment:
It's better to move all variable declarations to the top because the JavaScript interpreter will move them up anyway. See [this blog post](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html) on hoisting for a detailed explanation.